game.js

```diff
@@ -1,26 +1,49 @@
+// IRON
 const ironAmount = document.querySelector('#iron-amount');
-const fuelAmount = document.querySelector('#fuel-amount');
 const ironIncome = document.querySelector('#iron-income');
+// FUEL
+const fuelAmount = document.querySelector('#fuel-amount');
 const fuelIncome = document.querySelector('#fuel-income');
+// CHIPS
+const chipsAmount = document.querySelector('#chips-amount');
+const chipsIncome = document.querySelector('#chips-income');
+// BUTTONS
 const ironBtn = document.querySelector('#iron-btn');
 const fuelBtn = document.querySelector('#fuel-btn');
+const chipsBtn = document.querySelector('#chips-btn');
 
 let iron = 0;
 let miners = 1;
 let fuel = 0;
 let fuelers = 0;
+let chips = 10;
+let chipMakers = 0;
 
 ironBtn.addEventListener("click", () => {
     if (iron >= 5) {
-        iron -= 5;
-        miners++;
+        if (chips >= 1) {
+            iron -= 5;
+            chips -= 1;
+            miners++;
+        }
     }
 });
 
 fuelBtn.addEventListener("click", () => {
     if (iron >= 15) {
-        iron -= 15;
-        fuelers++;
+        if (chips >= 3) {
+            iron -= 15;
+            chips -= 2;
+            fuelers++;
+        }
+    }
+});
+
+chipsBtn.addEventListener("click", () => {
+    if (iron >= 150) {
+        iron -= 150;
+        chipMakers++;
+        chipsBtn.style.visibility = 'hidden';
```

Comment:
Hiding the button is sensible, but it might also be a good idea to check you don't already have a chip maker if the idea is to only allow one. Just in case the user is holding enter or something and it calls this function multiple times.